src/styles/App.sass

```
@@ -0,0 +1,3 @@
+.Main
```

```suggestion
.main
```
Nhưng k bổ sung thêm mấy breakpoints khác khi responsive à?

---

Chưa sửa ở đây này, class thông thường thì để in thường thôi. Còn nếu sử dụng styles component hay gì đó thì mới để in hoa như này nhé